README.md

```diff
@@ -1 +1,71 @@
-# Leaderboard
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet) 
+
+# Leaderboard 
+This leaderboard website displays scores submitted by different players. It also allows you to submit your score. All data is preserved thanks to the external Leaderboard API service.
+
+
+## Built With 🔨
+
+- HTML
+- CSS
+- Git & Github
+- Webpack
+- Jvascript
+- 
+
+### Install
+
+To get a local copy up and running follow these simple example steps.
+- Open terminal
+- Clone this project by the command: 
+
+```
+$ git clone git@github.com:Taher-web-dev/Leaderboard.git
+```
+
+- Then go to the main folder using the next command:
+
+```
+$ cd Leaderboard/
+```
+
+- Finally, install npm , then run <b> npm start </b> in your console.
+
+
+### Prerequisites
+
+- IDE to edit and run the code (We use Visual Studio Code 🔥).
+- Git to versionning your work.
+
+
+### Usage
+
+- Web developement practioner
+- For anyone who wants to create a display score leaderboard site.
+
+### Live Demo 
+[site]())
```

Comment:
[OPTIONAL]

- Please remove the `Live Demo` section or add the correct live link to make the README professional.